echopype/convert/parse_base.py

```diff
@@ -563,9 +563,11 @@ def _read_datagrams(self, fid):
 
             # IDX datagrams contain lat/lon and vessel distance from .idx files
             elif new_datagram["type"].startswith("IDX"):
+                self.idx["ping_number"].append(new_datagram["ping_number"])
+                self.idx["file_offset"].append(new_datagram["file_offset"])
                 self.idx["vessel_distance"].append(new_datagram["distance"])
-                self.idx["latitude_idx"].append(new_datagram["latitude"])
-                self.idx["longitude_idx"].append(new_datagram["longitude"])
+                self.idx["idx_latitude"].append(new_datagram["latitude"])
+                self.idx["idx_longitude"].append(new_datagram["longitude"])
```

Comment:
Since these are already under `self.idx` you could remove the idx_ here also.

Reply:
made this change 👍